cpp/proto/arcticc/pb2/storage.proto

```diff
@@ -184,7 +184,7 @@ message LibraryConfig {
     Portable library configuration which is sufficient to recreate a library
     */
     LibraryDescriptor lib_desc = 1;
-    map<string, VariantStorage> storage_by_id = 2;
+    map<string, StorageConfig> storage_by_id = 2;
```

Comment:
Is this going to be backwards compatible? We have literally thousands of these stored

Reply:
I think we must test (see my comment in the description).

Alternatively, we can simply ignore this PR.

Reply:
We use `FieldMask` in some internal code, so I think we might just have to live with this legacy name

Reply:
Can we just alias it in the C++ code? Nobody cares what it's called in the protobuf, and in the C++ we can call it whatever we want. The place for that would be in protobuf.hpp